scripts/wsrep_sst_xtrabackup-v2.sh

```diff
@@ -270,6 +270,51 @@ interruptable_timeout() {
     echo ${piperesult[@]}
 }
 
+# Monitor progress of SST and kill the process if stalled.
+monitor_sst_progress() {
+  local tmpsstdir=$1
+  shift
+  local pid=$1
+  shift
+  local timeout=$1
+  shift
+  currenttimeout=0
```

Comment:
These three variables should be local if possible.

Reply:
Also, it would be good to use underscore in the names, 
  currenttimeout -> current_timeout
  previoussize -> previous_size
to make it easier to read.